Components/button/button.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css">
+    <link rel="stylesheet" href="button.css">
+    <link rel="stylesheet" href="../avatar/avatar.css">
+    <title>Button</title>
+</head>
+<body>
+    <section>
+        <div class="container">
```

Comment:
I believe there is no need of adding a div here. You can add the class container to the section and it would work the same.

Reply:
Thats true but I like to structure it that way, thanks for the feedback though